src/chrome/content/rules/Hollywood-Reporter.xml

```diff
@@ -1,52 +1,40 @@
-
 <!--
-Disabled by https-everywhere-checker because:
-Fetch error: http://hollywoodreporter.com/ => https://hollywoodreporter.com/: Too many redirects while fetching 'https://hollywoodreporter.com/'
-Fetch error: http://www.hollywoodreporter.com/ => https://www.hollywoodreporter.com/: Too many redirects while fetching 'https://www.hollywoodreporter.com/'
-
-	For problematic rules, see Hollywood-Reporter-mismatches.xml.
-
-
-	CDN buckets:
-
-		- cs422.wac.thetacdn.net
-
-			- cdn[1-6].thr.com
-
-
-	Nonfunctional hosts in *hollywoodreporter.com:
-
-		- origin-www *
-
-	* 404
 
-
-	Problematic domains:
-
-		- in *hollywoodreporter.com:
-
-			- edit		(works, expired 2013-06-20)
-			- m		(expired, mismatched)
-
-		- thr[1-4].pgmcdn.net *
-
-	* Works, akamai
+	Problematic hosts in *hollywoodreporter.com:
+
+		- ^ (redirects to www over http)
```

Comment:
HTTPS Everywhere will handle it.

Reply:
I think they either changed it since I made the PR, or that's not what I meant. `^` now redirects to itself over http before redirecting to `www` over http.